bokehjs/src/lib/models/textures/image_url_texture.ts

```diff
@@ -30,6 +30,7 @@ export abstract class ImageURLTexture extends Texture {
   initialize(): void {
     super.initialize()
     this.image = new Image()
+    this.image.crossOrigin = "Anonymous"
```

Comment:
Unlike other places, this one has a capitalized "A" not sure if that is intentional / matters

Reply:
It doesn't matter, but this reminds me that the mechanism implemented in `ImageURL` should be generalized and used here and other places.